frontend_tests/node_tests/settings_profile_fields.js

```diff
@@ -81,7 +75,8 @@ function test_populate(opts) {
     assert.equal(num_appends, fields_data.length);
 }
 
-run_test("populate_profile_fields", () => {
+run_test("populate_profile_fields", (override) => {
+    override(settings_profile_fields, "create_sortable", () => {});
```

Comment:
I thought we had [agreed](https://github.com/zulip/zulip/pull/17439#issuecomment-790858737) that increased use of `__Rewire__` is the wrong direction, and yet this increases use of `__Rewire__`.